Lib/dataclasses.py

```diff
@@ -337,6 +341,24 @@ def _tuple_str(obj_name, fields):
     return f'({",".join([f"{obj_name}.{f.name}" for f in fields])},)'
 
 
+def _recursive_repr(user_function):
+    'Decorator to make a repr function return "..." for a recursive call'
```

Comment:
I don't think this merits a docstring, since it's a function internal to this module. Please just make this a comment. That's what's done elsewhere in this module.